src/shared.ts

```diff
@@ -121,7 +121,7 @@ const createWrapper = (stripeConnect: any) => {
             sdkVersion: version
           }
         }
-      });
+      }).connect;
```

Comment:
This is exposing an internal implementation/property - can we instead fix this on our repository by adding `setReactSdkAnalytics` to our public interface? Not adding private repo links here, but sent those over DM